package.json

```diff
@@ -69,9 +69,9 @@
     "typescript": "^4.7.3"
   },
   "peerDependencies": {
-    "jest": "^25.3.0 || ^26.0.1 || ^27.0.1 || ^28.0.2"
+    "jest": "^25.3.0 || ^26.0.1 || ^27.0.1 || ^28.0.2 || ^29.0.1"
   },
   "engines": {
-    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
+    "node": "^14.15.0 || >=16.0.0"
```

Comment:
```console
$ npm v jest@29.0.1 engines
{ node: '^14.15.0 || ^16.10.0 || >=18.0.0' }
```

Reply:
Wait so only LTS versions?
i.e. 15.x.x is not supported?

Reply:
We have run linting only on Node.js LTS. For example, see the [CI job log](https://github.com/stylelint/stylelint/runs/8188711914?check_suite_focus=true).

Node.js v15 [reached End-of-Life](https://github.com/nodejs/Release/#end-of-life-releases).